setup.py

```diff
@@ -42,5 +42,6 @@
           "Programming Language :: Python :: 3.8",
           "Programming Language :: Python :: 3.9",
           "Programming Language :: Python :: 3.10",
+          "Programming Language :: Python :: 3.11",
```

Comment:
Thanks! Could you please also *remove* 3.6?

Reply:
done!